src/components/details/MatchDetailsContent.jsx

```diff
@@ -14,7 +14,8 @@ const useStyle = makeStyles((_) => ({
     display: 'flex',
     flexDirection: 'column',
     alignItems: 'center',
-    paddingTop: '2vh'
+    paddingTop: '2vh',
+    marginRight: '5vh'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/FeNixCrew/MiEntradita-frontend/pullRequest?prid=8421134)
```suggestion
    marginRight: "5vh"
```